test/unit/test_assistant_v1.py

```diff
@@ -1,1576 +1,3973 @@
+
+##############################################################################
+# Start of Service: Message
+##############################################################################
+# region
+
+
+#-----------------------------------------------------------------------------
+# Test Class for message
+#-----------------------------------------------------------------------------
+class TestMessage():
+
+    #--------------------------------------------------------
+    # Test 1: Send fake data and check response
+    #--------------------------------------------------------
+    @responses.activate
+    def test_message_response(self):
+        body = self.construct_full_body()
+        response = fake_response_MessageResponse_json
+        send_request(self, body, response)
+        assert len(responses.calls) == 1
+
+    #--------------------------------------------------------
+    # Test 2: Send only required fake data and check response
+    #--------------------------------------------------------
+    @responses.activate
+    def test_message_required_response(self):
+        # Check response with required params
+        body = self.construct_required_body()
+        response = fake_response_MessageResponse_json
+        send_request(self, body, response)
+        assert len(responses.calls) == 1
+
+    #--------------------------------------------------------
+    # Test 3: Send empty data and check response
+    #--------------------------------------------------------
+    @responses.activate
+    def test_message_empty(self):
+        check_empty_required_params(self, fake_response_MessageResponse_json)
+        check_missing_required_params(self)
+        assert len(responses.calls) == 0
+
+    #-----------
+    #- Helpers -
+    #-----------
+    def make_url(self, body):
+        endpoint = '/v1/workspaces/{0}/message'.format(body['workspace_id'])
+        url = '{0}{1}'.format(base_url, endpoint)
+        return url
+
+    def add_mock_response(self, url, response):
+        responses.add(responses.POST,
+                      url,
+                      body=json.dumps(response),
+                      status=200,
+                      content_type='application/json')
+
+    def call_service(self, body):
+        service = AssistantV1(
+            authenticator=NoAuthAuthenticator(),
+            version='2019-02-28',
+        )
+        service.set_service_url(base_url)
+        output = service.message(**body)
+        return output
+
+    def construct_full_body(self):
+        body = dict()
+        body['workspace_id'] = "string1"
+        body.update({
+            "input":
+                MessageInput._from_dict(json.loads("""{"text": "fake_text"}""")
+                                       ),
+            "intents": [],
+            "entities": [],
+            "alternate_intents":
+                True,
+            "context":
+                Context._from_dict(
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['Context' may be undefined, or defined from star imports: ibm_watson.assistant_v1](https://app.codacy.com/manual/watson-developer-cloud/python-sdk/pullRequest?prid=4835642)